### PR TITLE
Social media buttons

### LINKED ISSUE #4094: Social sharing
add social sharing button bar add to market trading page initially. It will sit next to the copy and favorites buttons.

make a component for the buttons, we might add them to other places.

### PR DESCRIPTION
**Pull request by adrake33**
Closes #4094

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://featurepeek.com). 🌠

 This link will always point to this PR's latest changes: https://peek.run/l74h3xo

File: packages/augur-ui/src/modules/common/icons.tsx [hunk 1]
```diff
@@ -1091,6 +1091,20 @@ export const GoogleLogin = (
   </svg>
 );
 
+export const TwitterButton = (
+  <svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (phoebemirman) [inline packages/augur-ui/src/modules/common/icons.tsx]: you should remove the width, height, xlmns, and fills on path

Comment (phoebemirman) [reply packages/augur-ui/src/modules/common/icons.tsx]: then style it in css

File: packages/augur-ui/src/modules/market/components/common/social-media-buttons.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    const url = `https://twitter.com/intent/tweet?text=${encodedMarketDescription}&url=${encodedMarketUrl}`;
+    window.open(url, '', 'width=600,height=300');
+    return false;
+  }
+
+  render() {
+    const encodedMarketUrl = encodeURIComponent(`${window.location.origin}/#!/market?id=${this.props.marketAddress}`);
+    const encodedMarketDescription = encodeURI(this.props.marketDescription);
+    return (
+      <div>
```
Comment (phoebemirman) [inline packages/augur-ui/src/modules/market/components/common/social-media-buttons.tsx]: you prob don't need this div or could be <>

File: packages/augur-ui/src/modules/market/components/market-header/market-header.tsx [hunk 3]
```diff
@@ -190,7 +191,8 @@ export default class MarketHeader extends Component<
                   </button>
                   <MarketTypeLabel marketType={marketType} />
                 </WordTrail>
-                <div id="copy_marketId" data-clipboard-text={market.id}>
+                <SocialMediaButtons marketAddress={market.id} marketDescription={description} />
+                <div className={Styles.copy_marketId} id="copy_marketId" data-clipboard-text={market.id}>
```
Comment (phoebemirman) [inline packages/augur-ui/src/modules/market/components/market-header/market-header.tsx]: can class names be upper case? and also not sure if this needs its own class, could just do > div

**Review by phoebemirman — CHANGES_REQUESTED** [review]
spacing looks off, can you check the mock 
Screen Shot 2019-10-08 at 3 46 28 PM

File: packages/augur-ui/src/modules/market/components/common/social-media-buttons.tsx (hunk 2 above)
Comment (adrake33) [reply packages/augur-ui/src/modules/market/components/common/social-media-buttons.tsx]: Ok, will change it to <>

File: packages/augur-ui/src/modules/market/components/market-header/market-header.tsx (hunk 3 above)
Comment (adrake33) [reply packages/augur-ui/src/modules/market/components/market-header/market-header.tsx]: Yeah, I'll remove the class & update the CSS

**Comment by bthaile** [comment]
getting this error when trying to share. could be b/c I'm on localhost
image

twitter looks good

**Comment by adrake33** [comment]
@bthaile I got that on my local node too, but I think it's just because it has localhost in the URL. Other URLs do not give that error

**Comment by bthaile** [comment]
@adrake33  this PR has other PR commits. need to pull out only the social stuff.

**Comment by adrake33** [comment]
@bthaile Weird, ok I will remove those from this branch.

**Comment by bthaile** [comment]
seeing this 

image

**Comment by adrake33** [comment]
Oh right, well I guess we’ll both need to go to Specialtys?

**Comment by adrake33** [comment]
@bthaile I got the SVGs from Figma, but apparently the fillRule/clipRule weren't formatted correctly. Fixed.

**Comment by bthaile** [comment]
same thing happened to Andy. search for `fillRule` and `clipRule` to find an example to go with.

**Comment by adrake33** [comment]
It's fixed -- the issue was just that I was using fill-rule/clip-rule instead of fillRule/clipRule.

**Review by bthaile — APPROVED** [review]
container isn't really needed here but lets keep it, it might be needed in the future.

**Review by phoebemirman — APPROVED** [review]

**Pull request merged**